src/script/Config.ts

```diff
@@ -31,110 +31,65 @@ export const ACCENT_ID = {
   TURQUOISE: 6,
 };
 
-export class Configuration {
-  readonly APP_BASE = env.APP_BASE || 'https://app.wire.com';
-  readonly APP_NAME = env.APP_NAME || 'Webapp';
-  readonly APP_INSTANCE_ID = createRandomUuid();
-  readonly BACKEND_NAME = env.BACKEND_NAME || 'Wire';
-  readonly BACKEND_REST = env.BACKEND_REST || 'https://prod-nginz-https.wire.com';
-  readonly BACKEND_WS = env.BACKEND_WS || 'wss://prod-nginz-ssl.wire.com';
-  readonly BRAND_NAME = env.BRAND_NAME || 'Wire';
-  readonly COUNTLY_API_KEY = env.COUNTLY_API_KEY;
-  readonly ENABLE_DEV_BACKEND_API = env.ENABLE_DEV_BACKEND_API;
-  readonly ENVIRONMENT = env.ENVIRONMENT || 'production';
-  readonly FEATURE = {
```

Comment:
Note that now all the types and default values are generated server side. No need for this huge config file anymore